go/ql/src/experimental/CWE-321/HardcodedKeysLib.qll

```diff
@@ -57,7 +57,8 @@ module HardcodedKeys {
         pkg =
           [
             "github.com/golang-jwt/jwt/v4", "github.com/dgrijalva/jwt-go",
-            "github.com/form3tech-oss/jwt-go", "github.com/ory/fosite/token/jwt"
+            "github.com/form3tech-oss/jwt-go", "github.com/ory/fosite/token/jwt",
+            "github.com/gogf/gf-jwt/v2"
```

Comment:
Are there methods `Token.SignedString` or `SigningMethod.Sign` in `github.com/gogf/gf-jwt/v2`? I had a look and didn't see them, so I don't think this change does anything